applications/editor.lisp

```diff
@@ -1547,6 +1547,63 @@ If no such form is found, then return the CL-USER package."
 (defun beginning-of-top-level-form-command ()
   (beginning-of-top-level-form (current-buffer *editor*)))
 
+(defun newline-command ()
+  (insert (current-buffer *editor*)  "
+"))
+
+(defun open-line-command ()
+  (let ((buffer (current-buffer *editor*)))
+    (move-end-of-line buffer)
+    (newline-command)))
+
+(defun eval-expression-command ()
+  (print (eval (read-from-string (read-from-minibuffer "Eval: ")))))
+
+(defun copy-region-command ()
+  (kill-region-command)
+  (yank-command))
+
+(defvar *last-search-string* "")
```

Comment:
This should be bound in EDITOR-MAIN so that separate editor instances stay separate.